compiler/resolution/resolveFunction.cpp

```diff
@@ -45,6 +45,7 @@
 #include "stmt.h"
 #include "stringutil.h"
 #include "symbol.h"
+#include "Thunk.h"
```

Comment:
is this needed here?

Reply:
yes, for the various macros that do something over each AST node type (e.g., "for each child").